plugins/woocommerce/src/Enums/OrderStatus.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Enums;
+
+/**
+ * Enum class for all the order statuses.
+ *
+ * For a full documentation on the public order statuses, please refer to the following link:
+ * https://woocommerce.com/document/managing-orders/order-statuses/
+ */
+class OrderStatus {
```

Comment:
Nice! I'll suggest making it final; it's not getting extended by third-party extensions.

Reply:
Good call. Added the `final` keyword 👍